Makefile

```diff
@@ -418,84 +469,24 @@ ifeq ($(DISABLE_SERVICE_TLS), true)
 endif
 
 .PHONY: undeploy_bundle
-undeploy_bundle: operator-sdk
+undeploy_bundle: operator-sdk ## Undeploy deployed bundle.
 	- $(OPERATOR_SDK) cleanup $(OPERATOR_NAME)
 
-# Deploy a Cryostat instance
 .PHONY: create_cryostat_cr
-create_cryostat_cr: destroy_cryostat_cr
+create_cryostat_cr: destroy_cryostat_cr ## Deploy a Cryostat instance.
 	$(CLUSTER_CLIENT) create -f config/samples/operator_v1beta1_cryostat.yaml
 
 .PHONY: create_clustercryostat_cr
-create_clustercryostat_cr: destroy_clustercryostat_cr
+create_clustercryostat_cr: destroy_clustercryostat_cr ## Deploy a Cryostat cluster.
```

Comment:
I would describe this as "Deploy a cluster-wide Cryostat instance", and adjust the `destroy_` one accordingly too. It's still just one Cryostat instance in either case, there is just currently and API kind split between Cryostats deployed into a single k8s Namespace which they also monitor, vs Cryostats deployed into a Namespace and monitoring another list of Namespaces.